installer/pkg/components/server/deployment.go

```diff
@@ -130,13 +145,16 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 								Privileged: pointer.Bool(false),
 								RunAsUser:  pointer.Int64(31001),
 							},
-							Ports: []corev1.ContainerPort{{
-								Name:          ContainerPortName,
-								ContainerPort: ContainerPort,
-							}, {
-								Name:          PrometheusPortName,
-								ContainerPort: PrometheusPort,
-							}},
+							Ports: []corev1.ContainerPort{
```

Comment:
This is a tidy-up of code formatting

Reply:
Thank you for applying this formatting :)